osu.Game.Rulesets.Catch.Tests/TestSceneCatcher.cs

```diff
@@ -31,10 +31,23 @@ public class TestSceneCatcher : OsuTestScene
         [Resolved]
         private OsuConfigManager config { get; set; }
 
-        private Container<CaughtObject> droppedObjectContainer;
+        [Cached]
+        private readonly DroppedObjectContainer droppedObjectContainer;
+
+        private readonly Container trailContainer;
 
         private TestCatcher catcher;
 
+        public TestSceneCatcher()
+        {
+            Add(trailContainer = new Container
+            {
+                Anchor = Anchor.Centre,
+                Depth = -1
```

Comment:
It is not related to the change, but the depth was wrong (does not match gameplay) and I added this.

Reply:
Can you just combine these into a single `AddRange` call and reorder them, rather than using `Depth`? We generally avoid using `Depth` whenever possible.

Reply:
A `Catcher` is recreated and added in the setup step of each test.
But `DroppedObjectContainer` is `Cached` so it is added in constructor.
Because we need the consistent ordering between `Catcher` and `DroppedObjectContainer`, I used `Depth`.